text/0155-external-modules.md

```diff
@@ -0,0 +1,106 @@
+# Instrumentation Ecosystem Management
```

Comment:
Should this guidance also apply to other ecosystem components like exporters, propagators, resource detectors, etc.? Those things are also in the registry/contrib repos.

Reply:
They should indeed. Do you have suggestions for better title?

Reply:
I don't have any great suggestion, maybe just "Ecosystem Management"?

Reply:
changed name